.gitignore

```diff
@@ -162,3 +162,6 @@ RESTART/
 _dacegraphs
 
 **/testing/output/*
+
+*nc
+*png
```

Comment:
I think the `.nc` files are probably fine to ignore, but we may in the future (if not already) have `.png` files as part of the repository. So I would not ignore the png files.